cmd/capabilities.go

```diff
@@ -34,9 +34,8 @@ var printVersion bool
 // capabilitiesCmd represents the capabilities command
 var capabilitiesCmd = &cobra.Command{
 	Use:     "capabilities",
-	Aliases: []string{"c", "cap"},
```

Comment:
I'd removed that for tab completion. When "c" exists, `capabilities` subcommand is not presented on the tab completion suggestions when you are just typing `c` because the subcommand is matched by the `Aliases`.

Reply:
OK, let keep it this way then